pymc3/distributions/mixture.py

```diff
@@ -36,8 +36,34 @@ class Mixture(Distribution):
     w : array of floats
         w >= 0 and w <= 1
         the mixture weights
-    comp_dists : multidimensional PyMC3 distribution or iterable of one-dimensional PyMC3 distributions
-        the component distributions :math:`f_1, \ldots, f_n`
+    comp_dists : multidimensional PyMC3 distribution (e.g. `pm.Poisson.dist(...)`)
+        or iterable of one-dimensional PyMC3 distributions the
+        component distributions :math:`f_1, \ldots, f_n`
+
+    Example
+    -------
+    # 2-Mixture Poisson distribution
```

Comment:
This doesn't display correctly. Need to add the `code` tag.

Reply:
Just to check: should be `.. code-block:: python`?

Reply:
http://livesphinx.herokuapp.com/ is directionally helpful

Reply:
(that is to say, yes, that's it, but then it needs a newline after it, and possibly indentation.)